internal/config/mount_config.go

```diff
@@ -69,6 +76,15 @@ type MetadataCacheConfig struct {
 	// on the maximum number of type-cache entries,
 	// which are maintained at per-directory level.
 	TypeCacheMaxEntries int `yaml:"type-cache-max-entries,omitempty"`
+
+	// StatCacheMaxSizeMB is the maximum size of stat-cache
+	// in MiBs.
+	// It can also be set to -1 for no-size-limit, 0 for
+	// no cache. Values below -1 are not supported.
+	// Any value set below -1 will return an error.
```

Comment:
"Values below -1 are not supported. Any value set below -1 will return an error." -> Values below -1 are not supported.

Reply:
done